gcloud/test__helpers.py

```diff
@@ -184,9 +186,11 @@ class Test__get_default_service_project_id(unittest2.TestCase):
     config_file = 'config_default'
 
     def setUp(self):
-        import tempfile
         import os
-        self.temp_config_path = tempfile.gettempdir()
+        from gcloud._testing import _NamedTemporaryFile
+
+        with _NamedTemporaryFile() as temp:
+            self.temp_config_path = os.path.dirname(temp.name)
 
```

Comment:
Shouldn't all the stuff below should be inside the `with _NamedTemporaryFile() as temp:` block?  Otherwise, the file will be removed.